cql3/Cql.g

```diff
@@ -614,6 +614,7 @@ deleteOp returns [std::unique_ptr<cql3::operation::raw_deletion> op]
 
 usingClauseDelete[std::unique_ptr<cql3::attributes::raw>& attrs]
     : K_USING K_TIMESTAMP ts=intValue { attrs->timestamp = ts; }
+    | K_USING K_TIMEOUT to=term { attrs->timeout = to; }
```

Comment:
K_UPDATE K_THE K_DOCUMENTATION

Reply:
Oh, and USING TIMEOUT :timeout USING TIMESTAMP :ts should be supported.

Reply:
I mean, USING TIMEOUT :timeout AND TIMESTAMP :ts, I think.

Reply:
ah, right